roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -182,7 +182,6 @@
     openshift_logging_kibana_master_url: "{{ openshift_logging_master_url }}"
     openshift_logging_kibana_master_public_url: "{{ openshift_logging_master_public_url }}"
     openshift_logging_kibana_image_prefix: "{{ openshift_logging_image_prefix }}"
```

Comment:
We should be removing the prefixes too if we are going to defer to the ones defined per role